src/config/2021.json

```diff
@@ -386,6 +386,14 @@
       "github": "jonoalderson",
       "twitter": "jonoalderson"
     },
+    "jzyang": {
```

Comment:
FYI @jzyang adding your contributor info. Open a new PR to make any changes.

Reply:
Thanks @rviscomi !